integ-test/src/test/resources/correctness/queries/groupby.txt

```diff
@@ -54,4 +54,4 @@ SELECT OriginCountry, COUNT(FlightNum) AS cnt FROM opensearch_dashboards_sample_
 SELECT OriginCountry, COUNT(FlightNum) AS cnt FROM opensearch_dashboards_sample_data_flights GROUP BY OriginCountry HAVING 1 < cnt
 SELECT OriginCountry, COUNT(FlightNum) FROM opensearch_dashboards_sample_data_flights GROUP BY OriginCountry HAVING MIN(FlightDelayMin) > 100
 SELECT OriginCountry, COUNT(FlightNum) FROM opensearch_dashboards_sample_data_flights GROUP BY OriginCountry HAVING COUNT(FlightNum) > 1 OR MIN(FlightDelayMin) > 100
-SELECT OriginCountry, COUNT(FlightNum) FROM opensearch_dashboards_sample_data_flights GROUP BY OriginCountry HAVING COUNT(FlightNum) = 1 AND MIN(FlightDelayMin) <= 100
+SELECT OriginCountry, COUNT(FlightNum) FROM opensearch_dashboards_sample_data_flights GROUP BY OriginCountry HAVING COUNT(FlightNum) = 1 AND MIN(FlightDelayMin) <= 100
```

Comment:
Can you fix this whitespace error? An easy way to do this:
`git checkout upstream/main -- integ-test/src/test/resources/correctness/queries/groupby.txt`

Reply:
Ah thank you, I was wracking my brain trying to figure out what the difference between these two lines were, turns out that it is from the fact that at some point I had accidentally removed a newline at the end of the file

Reply:
fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/266/commits/8fd104b9427f2a73386eb47aa1dd69e494cb43cf